### ISSUE TITLE
After rebooting device, app can't see downloaded manga

### ISSUE THREAD

**Issue by Madaraki-chan** [mihonapp/mihon#107]
### Steps to reproduce

1. Add any manga with downloaded chapters to the library using extensions, or simply add manga from a local source to the library.
2. Reboot the device.
3. Open the application.
4. Try to open any of the downloaded chapters with the internet turned off.

### Expected behavior

Already downloaded chapters will be available to read.

### Actual behavior

When the Internet is connected, instead of opening already downloaded chapters, they are opened directly from the source (not opened at all in case of local storage). When the Internet is disconnected, it complains about unavailability of the source. 

### Mihon version

0.16.1

### Android version

Android 9

### Device

ZTE Blade 20 Smart

### Other details

If you use the "invalidate downloads index", the downloaded chapters stop showing up as such. If you change the Storage location to a random folder in the settings and then specify the previous folder, the downloaded chapters will be available again, but only until the next reboot of the device. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.1](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by ehsan18t** [comment]
True. I am facing this bug quite often and I don't even have to reboot the device. If my internet is disconnected the downloaded chapter are **not detected**(actually detected but doesn't show the sign). However, I can read the downloaded chapters.

**Comment by Madaraki-chan** [comment]
I also noticed that in the memory properties of the application appears item "External storage" with a value of "1 object" (I do not use SD cards, also this option is not present in Tachiyomi 0.14.7). After rebooting the device, this item disappears and the application stops seeing the device memory.

**Comment by TruePadawan** [comment]
Go to Settings > Advanced >  Invalidate downloads index

**Comment by Madaraki-chan** [comment]
> Go to Settings > Advanced > Invalidate downloads index
I'll just duplicate what I have in "Other details".

If you use the "invalidate downloads index", the downloaded chapters stop showing up as such. If you change the Storage location to a random folder in the settings and then specify the previous folder, the downloaded chapters will be available again, but only until the next reboot of the device.

**Comment by TruePadawan** [comment]
> > Go to Settings > Advanced > Invalidate downloads index
> 
> I'll just duplicate what I have in "Other details".
> 
> If you use the "invalidate downloads index", the downloaded chapters stop showing up as such. If you change the Storage location to a random folder in the settings and then specify the previous folder, the downloaded chapters will be available again, but only until the next reboot of the device.

oh sorry, didn't see that

**Comment by Smol-Ame** [comment]
This sounds more like a device/OS-related issue or maybe your "Mihon" folder location for the app's Storage location. If you are still experiencing this issue, refer to the Storage FAQ: https://mihon.app/docs/faq/storage#storage-location

Otherwise, closing this issue in favor of/superceded by #2030
